screening/screen.H

```diff
@@ -618,13 +646,24 @@ number_t chabrier1998 (const plasma_state_t<number_t>& state,
 template <typename number_t>
 AMREX_GPU_HOST_DEVICE AMREX_FORCE_INLINE
 number_t actual_screen(const plasma_state_t<number_t>& state,
-                     const scrn::screen_factors_t& scn_fac)
+                     const scrn::screen_factors_t& scn_fac,
+                     const bool& skip_check = false,
+                     const number_t& skip_threshold = 1.01_rt)
```

Comment:
Passing primitive types by value is cheaper, and `skip_threshold` doesn't need to be a dual number when we're calculating derivatives (same for the other `actual_screen()` overload below).
```suggestion
                     const bool skip_check = false,
                     const amrex::Real skip_threshold = 1.01_rt)
```